### PR TITLE
Fix failing BE unit test

### PR DESCRIPTION
**Pull request by gitromba**
fixed , sometimes failing unit tests

### CODE REVIEW

File: models/classes/Lists/DataAccess/Repository/RdfValueCollectionRepository.php [hunk 1]
```diff
@@ -395,11 +395,19 @@ private function getPersistence(): SqlPersistence
 
     private function extractLabel(ValueCollectionSearchRequest $searchRequest, iterable $labels, string $subject): ?string
     {
-        if ($labels[$subject][$searchRequest->getDataLanguage()]) {
+        if (
```
Comment (mike-ionut-mihai-sandu-tao) [inline models/classes/Lists/DataAccess/Repository/RdfValueCollectionRepository.php:398]:
```suggestion
if (!empty($labels[$subject][$searchRequest->getDataLanguage()])) {
            return $labels[$subject][$searchRequest->getDataLanguage()];
        }
```

File: models/classes/Lists/DataAccess/Repository/RdfValueCollectionRepository.php [hunk 2]
```diff
@@ -395,11 +395,19 @@ private function getPersistence(): SqlPersistence
...
+            isset($labels[$subject])
+            && isset($labels[$subject][$searchRequest->getDataLanguage()])
+            && $labels[$subject][$searchRequest->getDataLanguage()]
+        ) {
             return $labels[$subject][$searchRequest->getDataLanguage()];
         }
 
-        if ($labels[$subject][$searchRequest->getDefaultLanguage()]) {
+        if (
```
Comment (mike-ionut-mihai-sandu-tao) [inline models/classes/Lists/DataAccess/Repository/RdfValueCollectionRepository.php:406]:
```suggestion
        if (!empty($labels[$subject][$searchRequest->getDefaultLanguage()])) {
            return $labels[$subject][$searchRequest->getDefaultLanguage()];
        }
```

**Review by mike-ionut-mihai-sandu-tao — COMMENTED** [review]
lgtm

**Comment by mike-ionut-mihai-sandu-tao** [comment]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Comment by gitromba** [comment]
Recommended changes are applied. Thanks for the review @mike-ionut-mihai-sandu-tao.

**Review by mike-ionut-mihai-sandu-tao — APPROVED** [review]
- [x] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Pull request merged**